hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/initialization/BlocklistAccountCreator.java

```diff
@@ -123,9 +127,9 @@ public void createMissingAccounts() {
     private List<String> readFileLines() {
         final List<String> fileLines;
         try {
-            fileLines = readPrivateKeyBlocklist(blocklistResourceName);
+            fileLines = readPrivateKeyBlocklist(DEFAULT_BLOCKLIST_RESOURCE, blocklistResourcePath);
```

Comment:
readPrivateJeyBlocklist() should throw a customized exception when IOException thrown instead of logging the warning, so you can catch it here and handle it.

Reply:
Added custom `BlocklistNotFoundException` and split the `readPrivateKeyBlocklist()` method to two separate methods `readPrivateKeyBlocklistFromPath()` and `readPrivateKeyBlocklistFromResource()`, both of which can throw that exception.